### ISSUE TITLE
Force a new row fieldtyp

### ISSUE THREAD

**Issue by helloDanuk** [statamic/ideas#96]
The suggestion [here](https://github.com/statamic/cms/issues/1432) reminded me of a problem I stumbled upon every now and then in V2:

I wanted to group fields without creating a section right away, so rather build smaller groups of content and options within a section. I used fields that do not need the full width and had two problems:

- At the end of such a 'group', space remained free, but the next field should only appear on a new row. Especially to better convey what belongs together for people who are not tech-savvy. This can be an input field with a few options, for example.

This can be solved if the fields are arranged so that the entire width is always filled or the next field is too wide so that it automatically appears on a new line. However, this does not always appear logical, especially if the input of a field is only short. And it doesn't change anything for problem # 2:

- If fields with conditions are shown and hidden, the number of fields varies and it is difficult to build blueprints that fill the entire width in any case.

**Idea: A placholder / linebreak fieldtype would be useful, so that the next field always shows up on a new row.**

<img width="662" alt="Bildschirmfoto 2020-02-24 um 10 04 02" src="https://user-images.githubusercontent.com/8265902/75140305-e4db3d80-56ee-11ea-889f-101de8e98153.png">

This idea can go further if the line-break field can have a display title, so we could group these subsets under a name.

**Comment by helloDanuk** [comment]
The range-fieldtyp is an actual usecase. The readability would be slightly better, if the fields min, max and step would be on the same row.

<img width="1034" alt="range fieldtype (feature request)" src="https://user-images.githubusercontent.com/8265902/78880713-1701ee00-7a56-11ea-9c9e-db4c4e762cf6.png">

**Comment by jackmcdade** [comment]
Yep, I thought of this request when i was arranging that layout with no good alternative. I still don't love this solution though, making an empty fieldtype just for alignment purposes feels like a `spacer.gif`. Hoping another idea pops into my head soon :)

**Comment by helloDanuk** [comment]
You're right, an empty fieldtype just for alignment purposes feels bad. Maybe somewhere a 'force new row' toggle option in every fieldsets (with a width narrower than 100%)? Like you did it with sets inside a bard 'included in output' for example.

But on the other hand, this feature would usually be requested from time to time only. So it should not distract the normal workflow.
